app/models/lockstep/payment_applied.rb

```diff
@@ -0,0 +1,16 @@
+class Lockstep::PaymentApplied < Lockstep::ApiRecord
+  self.model_name_uri = "v1/payments-applied"
+  self.id_ref = "payment_applied_id"
+  self.query_path = ""
+  load_schema(Schema::PaymentApplied)
+
+  def self.merge_all_attributes(objects, response)
```

Comment:
why is this method coded here? can you let me know if this has any difference apart from the general methods which are already in ApiRecord.rb

Reply:
in create method of `ApiRecord.rb` body was being sent inside an array but for this call we don't need that, Platform will raise exception if we do so